### PR TITLE
FIX: anonymize info when missing fields

### PR DESCRIPTION
**Pull request by kingjr**
`anonymized()` throw me an error on some edf files whose file_id and meas_id fields where missing

### CODE REVIEW

File: mne/io/meas_info.py [hunk 1]
```diff
@@ -1577,6 +1577,10 @@ def anonymize_info(info):
         del info['subject_info']
     info['meas_date'] = [0, 0]
     for key_1 in ('file_id', 'meas_id'):
+        if (key_1 not in info.keys()) or (info[key_1] is None):
```
Comment (Eric89GXL) [inline mne/io/meas_info.py]: `if info.get(key_1) is None`

File: mne/io/meas_info.py [hunk 2]
```diff
@@ -1577,6 +1577,10 @@ def anonymize_info(info):
...
+            continue
         for key_2 in ('secs', 'msecs', 'usecs'):
+            if key_2 not in info[key_1].keys():
```
Comment (Eric89GXL) [inline mne/io/meas_info.py]: `if key_2 not in info`

Comment (agramfort) [inline mne/io/meas_info.py]: @kingjr this is evil. You make a list for nothing and testing if key present becomes O(n) operation instead of O(1).

**Comment by kingjr** [comment]
@agramfort still evil?

**Comment by Eric89GXL** [comment]
All better. Thanks for making those changes so that @agramfort and I can sleep at night

**Comment by agramfort** [comment]
I am back in peace...

**Pull request merged**